.sync.yml

```diff
@@ -7,18 +7,14 @@
   unmanaged: true
 
 .travis.yml:
+  unmanaged: true
   branches:
```

Comment:
These two lines can be removed as .travis.yml is unmanaged.

Reply:
Without this line when I do a pdk update all the spec and acceptance testing sections of .travis.yml get nuked.

Reply:
Sorry I mean lines https://github.com/puppetlabs/puppetlabs-translate/pull/50/files#diff-f620401c6bdf661e1275316a6e01b62fR11-R12 . Since .travis.yml is unmanaged these are effectively not doing anything.

Reply:
got it